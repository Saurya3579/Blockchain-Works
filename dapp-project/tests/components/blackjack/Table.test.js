import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';

import Table from '../../../src/components/blackjack/Table';

test('should render NoMatch component', () => {
    const wrapper = shallow(<Table />);
    expect(toJSON(wrapper)).toMatchSnapshot();
});