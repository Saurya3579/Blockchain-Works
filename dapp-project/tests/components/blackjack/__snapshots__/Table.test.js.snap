exports[`test should render NoMatch component 1`] = `
<div
  className="table-board">
  <Hand
    hand={Array []}
    showDeck={true} />
  <hr />
  <Interface
    deal={[Function]}
    dealerscore={0}
    hit={[Function]}
    playerscore={0}
    stand={[Function]}
    status={null} />
  <hr />
  <Hand
    hand={Array []} />
</div>
`;
